frontend/js/stats.js

```diff
@@ -179,6 +193,146 @@ const fillRunTab = async (selector, data, parent = '') => {
+
+            const flowLabel = logEntry.flow ?
+                `<div class="ui green label" data-tooltip="Flow this command belongs to" data-position="top center"><i class="sitemap icon"></i> ${escapeString(logEntry.flow)}</div>` : '';
+
+            const idLabel = `<div class="ui label" data-tooltip="Unique identifier for this log entry" data-position="top center"><i class="hashtag icon"></i> ID: ${escapeString(logEntry.id)}</div>`;
+
+            const stdoutContent = logEntry.stdout ?
+                stdoutTemplate.replace('{{stdout}}', escapeString(logEntry.stdout)) : '';
+
+            const stderrContent = logEntry.stderr ?
+                stderrTemplate.replace('{{stderr}}', escapeString(logEntry.stderr)) : '';
+
+            // Add operation label for exceptions with specific tooltips
+            let operationLabel = '';
+            if (logEntry.type === 'exception') {
+                let operationTooltip;
+                switch (logEntry.cmd) {
+                    case 'run_scenario':
+                        operationTooltip = 'Exception occurred during main scenario execution runtime';
+                        break;
+                    case 'post_process':
+                        operationTooltip = 'Exception occurred during cleanup and post-processing phase';
+                        break;
+                    default:
+                        operationTooltip = `Exception occurred during ${logEntry.cmd} operation`;
+                }
+                const operationTitle = escapeString(logEntry.cmd.replace('_', ' ').replace(/\b\w/g, l => l.toUpperCase()));
+                operationLabel = `<div class="ui red label" data-tooltip="${operationTooltip}" data-position="top center"><i class="cogs icon"></i> ${operationTitle}</div>`;
+            }
+
+            // For exceptions, don't show the command section since it's now a label
+            const commandContent = logEntry.type === 'exception' ? '' :
+                `<div class="content">
+                    <h5 class="ui header"><i class="terminal icon"></i> Command</h5>
+                    <div class="ui segment">
+                        <code>${escapeString(logEntry.cmd)}</code>
+                    </div>
+                </div>`;
+
+            contentHTML += logCardTemplate
+                .replace('{{typeLabel}}', typeLabel)
+                .replace('{{flowLabel}}', flowLabel)
+                .replace('{{phaseLabel}}', phaseLabel)
+                .replace('{{operationLabel}}', operationLabel)
+                .replace('{{idLabel}}', idLabel)
+                .replace('{{commandContent}}', commandContent)
+                .replace('{{stdoutContent}}', stdoutContent)
+                .replace('{{stderrContent}}', stderrContent);
+        });
+
+        accordionHTML += containerTemplate
+            .replace('{{containerName}}', escapeString(containerName))
```

Comment:
can any of the replacements be there more than once every?

Reply:
double-checked and is correct.